index.js

```diff
@@ -284,11 +286,11 @@ RadixRouter.prototype = {
       var section = sections[i]
       node = node.children[section]
       if (!node) {
-        break
+        return success
```

Comment:
Early return. There was no sense in performing the check below because we already knew that it was going to evaluate to false. We had already done the check right here.